ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -197,8 +198,8 @@ public void sendRequest(
       requestBuilder.method(method, multipartBuilder.build());
     } else {
       // Nothing in data payload, at least nothing we could understand anyway.
-      // Ignore and treat it as if it were null.
-      requestBuilder.method(method, null);
+      // Creating empty requestBody to solve issue #3371
+      requestBuilder.method(method, RequestBody.create(null, ByteString.EMPTY));
```

Comment:
From looking up a few lines, it looks like there is a RequestBodyUtil.getEmptyBody which basically does this but might be a better call.  Then you won't have to add the import as it will hide behind something we already do.